### DISCUSSION TITLE
How to get return value from create() function

### DISCUSSION THREAD

**Discussion by seegithub** [spatie/db-dumper#204] [Q&A]
Hi, 

I am very grateful for the package which really helped me in backing up the database, but I want to ask if we can get the return value from the created() function, because I tried to fill in the wrong database name but the return value from the created() function just null, just like when the backup process was successful, thank you very much

my code is:
`$result = PostgreSql::create()->setDbName(env('DB_PREFIX') . $scope)->setUserName(env('DB_USERNAME'))->setPassword(env('DB_PASSWORD'))->useCompressor(new GzipCompressor)->dumpToFile($tempFile); var_dump($result);`

**Comment by danstoian07** [comment]
I am getting a blank page. My code is:

```
MySql::create()
            ->setDbName(env('DB_DATABASE'))
            ->setUserName(env('DB_USERNAME'))
            ->setPassword(env('DB_PASSWORD'))
            ->dumpToFile(env('DB_DATABASE').'.sql');
```

mysqldump is installed

```
~$ mysqldump
Usage: mysqldump [OPTIONS] database [tables]
OR     mysqldump [OPTIONS] --databases DB1 [DB2 DB3...]
OR     mysqldump [OPTIONS] --all-databases
OR     mysqldump [OPTIONS] --system=[SYSTEMOPTIONS]]
For more options, use mysqldump --help
```
